examples/flutter_gallery/lib/demo/flexible_space_demo.dart

```diff
@@ -124,10 +124,25 @@ class FlexibleSpaceDemoState extends State<FlexibleSpaceDemo> {
           ],
           flexibleSpace: new FlexibleSpaceBar(
             title : new Text('Ali Connors'),
-            background: new AssetImage(
-              name: 'packages/flutter_gallery_assets/ali_connors.png',
-              fit: ImageFit.cover,
-              height: _appBarHeight
+            background: new Stack(
+              children: <Widget>[
+                new AssetImage(
+                  name: 'packages/flutter_gallery_assets/ali_connors.png',
+                  fit: ImageFit.cover,
+                  height: _appBarHeight
+                ),
+                // This gradient ensures that the toolbar icons are distinct
+                // against the background image.
+                new DecoratedBox(
```

Comment:
I'm not sure, but I'm leaning towards not a feature. Using a gradient here makes sense because our image needs it. If the background was a solid color that contrasted nicely with the white toolbar icons, then the gradient would be out of place. It's easy for the developer to add a gradient and I think that deciding if its needed should be their responsibility.

We could add a flag to FlexibleSpaceBar that did this for you, but maybe its better to shy away from that kind of thing. I am leery of handy-dandy :-).